jupyterhub/templates/proxy/deployment.yaml

```diff
@@ -135,6 +135,8 @@ spec:
           livenessProbe:
             initialDelaySeconds: {{ .Values.proxy.chp.livenessProbe.initialDelaySeconds }}
             periodSeconds: {{ .Values.proxy.chp.livenessProbe.periodSeconds }}
+            timeoutSeconds: {{ .Values.proxy.chp.livenessProbe.timeoutSeconds }}
+            failureThreshold: {{ .Values.proxy.chp.livenessProbe.failureThreshold }}
```

Comment:
This value doesn't exist in the values file, at least not with a default like the readinessProbe:

https://github.com/jupyterhub/zero-to-jupyterhub-k8s/blob/0.11.1/jupyterhub/values.yaml#L194

(same on `main`)

Should that be added to the values file with a default like the readiness probe or at least be defined in the schema?